src/main/java/entity/Powerup.java

```diff
@@ -101,13 +102,10 @@ private void pickup(final Player p) {
 			case PIERCING:
 				p.setPiercing(NEW_PIERCING_LEVEL);
 				break;
-			case MINIGUN:
+			default:
```

Comment:
is het misschien een idee als de default niet hetzelfde wordt als een daadwerkelijk optie, maar bijvoorbeeld logt dat er iets mis is gegaan en dan wel de minigun equipt en als normaal doorgaat. dan hebben we denk ik wat meer overzicht over of de speler gewoon tegen de powerup aangelopen is of dat er echt iets fout is gegaan.